ddl/get.lua

```diff
@@ -115,21 +129,21 @@ local function get_space_schema(space_name)
 end
 
 local function prepare_sharding_func_for_call(space_name, sharding_func_def)
-    if type(sharding_func_def) == 'string' then
+    if type(sharding_func_def) == 'table' then
         local sharding_func = utils.get_G_function(sharding_func_def)
         if sharding_func ~= nil and
            ddl_check.internal.is_callable(sharding_func) == true then
             return sharding_func
         end
     end
 
-    if type(sharding_func_def) == 'table' then
-        local sharding_func, err = loadstring('return ' .. sharding_func_def.body)
-        if sharding_func == nil then
-            return nil, string.format(
-                "Body is incorrect in sharding_func for space (%s): %s", space_name, err)
-        end
-        return sharding_func()
+    if type(sharding_func_def) == 'function' then
+        return sharding_func_def
+    end
+
+    -- error from cache
+    if type(sharding_func_def) == 'string' then
+        return nil, sharding_func_def
     end
```

Comment:
We can remove this function at all and put already prepared sharding function to the cache. What do you think?

Reply:
We cannot because sharding function, defined by name, may not be exists at runtime (e.g deleted by user). So we need this checks: utils.get_G_function( ...

Reply:
(It seems, it is not actual anymore. Resolving the discussion.)